assets/app/widgets/suse-modal/suse-modal.directive.jade

```diff
@@ -0,0 +1,9 @@
+.modal-content
+    .modal-header
+        button.close(type='button', ng-click='dismiss()', aria-label='Close')
+            span(aria-hidden='true') ×
+        h4#suseModalLabel.modal-title {{ error.title }}
```

Comment:
Please keep in mind that more than one suse-modal could be invoked in the same page. Is an Id going to be enough?
Also, one way data binding?